scripts/SANS/isis_reduction_steps.py

```diff
@@ -1066,13 +1074,13 @@ def view(self, instrum):
         instrum.load_empty(wksp_name)
 
         # apply masking to the current detector
-        self.execute(None, wksp_name, instrum)
+        self.execute(None, wksp_name)
```

Comment:
hmm, that does not explain how we get away with passing in three parameters.  I checked and the base object has an execute with three [parameters](https://github.com/mantidproject/mantid/blob/master/scripts/reducer_singleton.py#L31:L38). It would default to it, but it should then throw an exception. Might need more investigation.